src/server/handlers/launchpad.js

```diff
@@ -480,16 +480,9 @@ const internalFindSnaps = async (owner, token) => {
     }
     return snaps;
   } catch (error) {
-    // At least for the moment, we just wrap the error we get from
-    // Launchpad.
-    const text = await error.response.text();
-    throw new PreparedError(error.response.status, {
-      status: 'error',
-      payload: {
-        code: 'lp-error',
-        message: text
-      }
-    });
+    // At least for the moment, we just throw the error and let it be caught
+    // later and sent in the response
+    throw error;
```

Comment:
`try { foo; } catch (error) { throw error; }` is equivalent to just `foo;`, so you might as well remove the `try` block.

Otherwise this is fine.

Reply:
Ah right, I even wanted to do that and then forgot.
Thanks for catching it.